features/ALB002_Alignment-layout.feature

```diff
@@ -1,6 +1,7 @@
 @implementer-agreement
 @ALB
 Feature: ALB002 - Alignment Layout
+The rule verifies the behavior of IfcAlignment and its segments, particularly in terms of nesting.
```

Comment:
In another PR, https://github.com/buildingSMART/ifc-gherkin-rules/pull/43 there is another description. We'll have to choose which one to use. Or is your description a generalization of the one below?
https://github.com/buildingSMART/ifc-gherkin-rules/blob/4db4722af5d76dbfc53495230f25d1cf80f6237e/features/ALB002_Alignment-layout.feature#L4-L7

Reply:
Not really, I've just missed that PR, sorry for that. 
I'll change my PR to have what's mentioned in yours!